docs/content/docs/dev/table/config.md

```diff
@@ -96,6 +124,11 @@ Flink SQL> SET 'table.exec.mini-batch.size' = '5000';
 {{< /tab >}}
 {{< /tabs >}}
 
+**Note:** All of the following configuration options can also be set globally in 
```

Comment:
Just a typography comment here: Use the hints here: https://github.com/apache/flink/tree/master/docs#info-hints

Reply:
Thank you!